python_utils.py

```diff
@@ -89,19 +89,20 @@ def get_args_of_function(function_node, args_to_ignore):
             a.id for a in function_node.args.args if a.id not in args_to_ignore]
 
 
-def open_file(filename, mode, encoding='utf-8'):
+def open_file(filename, mode, encoding='utf-8', newline=None):
```

Comment:
Should we make `''` as a default value? Considering we are only using that as of now (not sure!)

Reply:
+1

Reply:
We can't use `''` as a default value because `byte('b')` mode does not allow the use of newline argument with it and we are using `byte` mode at several places in codebase.